src/rest.js

```diff
@@ -85,6 +89,13 @@ export default class RestClient extends Vue {
     return this.user;
   }
 
+  async fetchOauth(redirect = window.location.href, list = true) {
+    const resp = await this.get('oauth/provider', {
```

Comment:
For Girder servers that do not have the OAuth plugin enabled, this will return some sort of 4xx response, so we should wrap this line in a try/catch, and if we detect an exception we can `return []`.